Cargo.toml

```diff
@@ -43,8 +43,8 @@ dirs-next = "2.0.0"
 dunce = "1.0.2"
 gethostname = "0.2.3"
 # Addresses https://github.com/starship/starship/issues/4251
-git-features = { version = "0.22.2", features = ["fs-walkdir-single-threaded"] }
-git-repository = "0.21.1"
+git-features = { version = "0.22.1", features = ["fs-walkdir-single-threaded"] }
```

Comment:
Did you mean to downgrade this? If possible, please also update this to 0.22.3.

Reply:
A great catch, that was unintentional. The latest version refers to `sha1` now, and it's good to see to see your PR landing here.